apps/studio/components/ui/DotGrid.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { motion } from 'framer-motion'
+
+interface DotGridProps {
+  rows: number
+  columns: number
+  count: number
+}
+
+const DotGrid: React.FC<DotGridProps> = ({ rows, columns, count }) => {
```

Comment:
in general let's avoid using FC, but use PropsWithChildren instead ([ref](https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13))

also in this case since we're not expecting a children, just DotGridProps will do